src/components/Bookmark.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+function Bookmark() {
+  const [bookmarkNotActiv, setBookmarkActiv] = useState(false)
```

Comment:
The state and the function which updates the state should always be named the same, like `data, setData`, etc.
How about:
```suggestion
  const [active, setActive] = useState(false)
```
Please try to avoid "doppelte Verneinung" when naming variables 🙂